src/components/map/battery-charge.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../resources/scss/variables";
+
+.zap-icon {
+    height: 32px;
+    width: 25px;
+    
+}
+
+.full-battery {
```

Comment:
This should be nested inside `.zap-icon` like so:

```scss
.zap-icon {
  .full-battery {
    ...
  }
}
```

(Yes, SCSS can do that)

This will cause SCSS to generate scoped CSS which avoids class name clashes.

Reply:
Corrected the nesting and also the indention size in 0ae3b810d687aac6ab93fb46d4f0b597984a06a5.